### PR TITLE
Feat/private game stats/bk

### PR DESCRIPTION
**Pull request by bkchilidawg**
## Description

Please include a description of what was changed

## Type of change

- [ ] fix
- [X] feat
- [X] test
- [ ] refactor
- [ ] docs

## Checklist

- [X] code has been self reviewed
- [X] code runs without any errors
- [X] thorough testing has been implemented if adding feature
- [X] all tests pass

### Thanks!

### CODE REVIEW

**Comment by mbkuhl** [comment]
Can we comment out the db:seed in the procfile so we can start to get some persistent data? I don't think we're making too many changes to the db going forward.

File: ''/cache/ruby-lsp-0.14.5.gem
Comment (mbkuhl) [inline ''/cache/ruby-lsp-0.14.5.gem:1]: Maybe we should also gitignore the cache? If it's easy do it, if not, leave it in

Comment (bkchilidawg) [reply ''/cache/ruby-lsp-0.14.5.gem:1]: unsure about cache but fixed the proc!

File: app/models/user.rb [hunk 1]
```diff
@@ -177,4 +177,58 @@ def top_three_finishes_competitive
...
+    end
+
+    rank
+  end
+
+  def total_overall_score_private_games(game_id)
+    game = games.find_by(id: game_id)
+    return 0 unless game
+
+    total_score = game.votes.where(user_id: id).sum(:score)
```
Comment (mbkuhl) [inline app/models/user.rb:211]: Nice elegant query

**Pull request merged**